Where_Is_My_Wife/Assets/_Main/Scripts/Animation/AnimationController.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using UniRx;
+using UnityEngine;
+using WhereIsMyWife.Managers;
+using Zenject;
+
+namespace WhereIsMyWife.Controllers
+{
+    public class AnimationController : MonoBehaviour
+    {
+        [SerializeField] private Animator _animator;
+
+        [Inject] private IMovementStateEvents _movementStateEvents;
+        [Inject] private IWallHangStateEvents _wallHangStateEvents;
+        [Inject] private IDashStateEvents _dashStateEvents;
+        [Inject] private IPlayerStateIndicator _stateIndicator;
```

Comment:
Change these to direct calls to a manager that needs to be turnned into a singleton, like `PlayerManager.DoSomething()` or `PlayerManager.GetSpeed()`

Reply:
da438a0813a32f4a61ffd9f71498940f5bb073a7